xmtp_mls/src/storage/encrypted_store/outbound_welcome_message.rs

```diff
@@ -0,0 +1,18 @@
+use super::schema::outbound_welcome_messages;
```

Comment:
If you update from `main` this file should now be gone, as well as the associated table

Reply:
oo i probably screwed something up in the merge, good catch